wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -54,7 +54,8 @@ <h1 class="w-header__title" id="header-title">
                 {{ search }}
             {% endif %}
         </div>
-        <div class="{% block right_column_classname %}right{% endblock %}">
+        <!-- no longer supported in the release notes.
```

Comment:
This looks like it will comment out the entire section - did you test this - I do not think this will work?

The goal, as per the issue, was to simply remove the block tags.

e.g.
```
<div class="right">
```